definitions.md

```diff
@@ -0,0 +1,62 @@
+# Definitions
+
+## Application Server
+
+This is the server that hosts the application.
+
+## End User Application
+
+This is the application used by the end user to connect to the application server and to interact with it.
+
+## Message
```

Comment:
A message can also refer to a message used to communicate with android services. Therefore we'll use Push Message instead.